src/deploy/D3MInit.sol

```diff
@@ -323,4 +362,13 @@ library D3MInit {
         plan.file("barb", compoundCfg.barb);
     }
 
+    function initOperatorPlan(
```

Comment:
@hexonaut shouldn't we rely the `D3MMom` here?
I see it is not happening in the other plans, so not sure.

Reply:
It's done in `initCommon` function.